src/rwe/io/tdf/NetSchemaTdfAdapter.cpp

```diff
@@ -0,0 +1,35 @@
+#include "NetSchemaTdfAdapter.h"
+// Special case parsing, only checks if there's a schema with type "NETWORK" (when searching all maps, a full parse and validity check is expensive)
+
+namespace rwe
+{
+    void NetSchemaTdfAdapter::onStart()
+    {
+        bHasNetSchema = false;
+        bInSchema = false;
```

Comment:
For next time: RWE doesn't use hungarian notation style, I'd prefer to drop the `b` prefix on these.